integration-test/src/main.rs

```diff
@@ -0,0 +1,173 @@
+extern crate core;
```

Comment:
The `extern crate core` is not needed any more, right? (Particularly given this cargo project has `edition` set)

Also, if you decide to add Clippy to CI, you may also want to enable pedantic etc, for parity with what we use elsewhere.

```suggestion
#![warn(clippy::pedantic)]
#![warn(unused_crate_dependencies)]
```

Reply:
Resolved in [8dfae92](https://github.com/heroku/heroku-java-metrics-agent/pull/66/commits/8dfae921c635b33ffd3d55145e4257835c8e4e82).